### ISSUE TITLE
Error when i use --no-check flag on Deno 

### ISSUE THREAD

**Issue by lowray** [garronej/evt#17]
Hello i'm trying to use Deno with https://github.com/asos-craigmorten/opine but i'm getting an error from your package.
The error is 
```
error: Uncaught SyntaxError: The requested module './CtxLike.ts' does not provide an export named 'CtxLike'
export { CtxLike } from "./CtxLike.ts";
~~~~~~~
    at <anonymous> (<https://deno.land/x/evt@v1.9.11/lib/types/interfaces/index.ts>:5:1)
```
I hope you fix this issue
Thanks!

**Comment by garronej** [comment]
@lowray  I am on it!

**Comment by garronej** [comment]
Fixed in v1.9.12
